sb-chat-history-server/src/lib/MSSQL/poolManager.js

```diff
@@ -1,24 +1,25 @@
 module.exports = class MSSQLPoolManager {
   constructor(env, mssql) {
     const {
-      DB_HOST, DB_USER, DB_PASSWORD, DB_NAME, DB_PORT
+      DB_HOST, DB_USER, DB_PASSWORD, DB_NAME, DB_PORT, DB_DOMAIN
     } = env;
-    console.log(env)
+    console.log(env);
     this.mssql = mssql;
     this.host = DB_HOST;
     this.user = DB_USER;
+    this.domain = DB_DOMAIN === '' ? undefined : DB_DOMAIN;
     this.password = DB_PASSWORD;
     this.databaseName = DB_NAME;
     this.databasePort = DB_PORT;
     this.mssql = mssql;
   }
 
   async connect() {
-    let port = this.databasePort
+    let port = this.databasePort;
     // process.env will set this to a string in some cases this is to handle
     try {
-      port = parseInt(this.databasePort, 10)
-    } catch(e) {
+      port = parseInt(this.databasePort, 10);
+    } catch (e) {
```

Comment:
not sure what happened here but we shouldn't have a catch that catches nothing

can you just change this to 

```
const port = parseInt(this.databasePort, 10);
```

if it throws, sobeit because it should cause a failure

(i know you didn't introduce this code but it is particularly bad)